Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.68.5"
+version="0.69.0"
```

Comment:
Maybe wait with a release until after the DMS is also fixed? Is this urgent?

And if this is a release, shouldn't it be `0.68.6`?

Reply:
I agree, let's hold version until DMS is fixed.